### DISCUSSION TITLE
Initialize has vanished

### DISCUSSION THREAD

**Discussion by ASCine** [thebaselab/codeapp#1030] [Q&A]
I am totally new to CodeApp, so this might be a stupid question but I have to ask.
I set up my GitHub and wanted to initialize my repo... so I tap on the third icon on the left bar and there it is. But then I accidentally didn't hit the button and now it is gone. I don't see a way to get back... ?!

Thanks for any advice.
Note: restart the app has no effect

**Comment by bummoblizard** [comment]
You have created an empty repository at Documents. To delete it, enable show hidden files in settings menu and delete the .git folder using the file explorer.

Use the clone feature to clone an existing GitHub repository.
